R/mc6_mthds.R

```diff
@@ -20,62 +20,47 @@
-#'   series which have borderline activity, i.e., series where the top 
-#'   parameter of the winning model was greater than or equal to 0.8*cutoff 
-#'   and less than or equal to 1.2*cutoff.}
-#'   \item{overfit.hit}{The overfit.hit flag recalculates the model winner 
-#'   after applying a small sample correction factor to the AIC values. If the 
-#'   hit-call would be changed after applying the small sample correction 
-#'   factor the series is flagged. Series with less than 5 concentrations where
-#'   the hill model won and series with less than 7 concentrations where the 
-#'   gain-loss model won are automatically flagged.}
-#'   \item{efficacy.50}{The efficacy.50 flag identifies concentration series 
-#'   with efficacy values (either the modeled top parameter for the winning
-#'   model or the maximum median response) less than 50 for percent activity
-#'   data or log2(1.5) for fold induction data}
-#'   \item{ac50.lowconc}{The ac50.lowconc flag identifies concentration series 
-#'   with ac50 values less than the minimum tested concentration.}
-#'   \item{viability.gnls}{The viability.gnls flag identifies concentration series 
-#'   of cell viability assays which were fit with gain-loss as the winning 
-#'   model and the series had an active hit-call.}
+#'   \item{noise}{Flag series as noisy if the quality of fit as calculated by the root mean square 
+#'   error (rmse) for the series is greater than the cutoff (coff); rmse > coff.}
+#'   \item{border}{Flag series if borderline activity is suspected based on modeled top parameter 
+#'   (top) relative to cutoff (coff); |top| <= 1.2(coff) or |top| >= 0.8(coff).}
+#'   \item{overfit.hit}{Method not yet updated for tcpl implementation. Flag hit-calls that would 
+#'   get changed after doing the small N correction to the aic values.}
+#'   \item{efficacy.50}{Flag low efficacy hits if series has an active hit call (hitc >= 0.9) and 
+#'   efficacy values (e.g. top and maximum median response) less than 50 percent; intended for 
+#'   biochemical assays. If hitc >= 0.9 and coff >= 5, then flag when top < 50 or max_med < 50. 
+#'   If hitc >= 0.9 and coff < 5, then flag when top < log2(1.5) or max_med < log2(1.5).}
+#'   \item{ac50.lowconc}{Flag series with an active hit call (hitc >= 0.9) if AC50 (concentration 
+#'   at 50 percent maximal response) is less than the lowest concentration tested;if hitc >= 0.9 
+#'   and ac50 < 10^logc_min, then flag.}
+#'   \item{viability.gnls}{Flag series with an active hit call (hitc >= 0.9) if denoted as cell 
+#'   viability assay with winning model is gain-loss (gnls); if hitc >= 0.9, modl=="gnls" and 
+#'   cell_viability_assay == 1, then flag.}
+#'   \item{medresp.3bmad}{Flag series where no median response values are greater than baseline as 
+#'   defined by 3 times the baseline median absolute deviation (bmad); nmed_gtbl = 0, where 
+#'   nmed_gtbl is the number of medians greater than 3*bmad.}
```

Comment:
`medresp.3bmad` has now been merged to main under the name `no.med.gt.3bmad`. Description here matches what is in the database, so just an update to the name is needed